docs/src/components/DocApi.vue

```diff
@@ -132,6 +134,46 @@ export default {
       }
 
       this.currentTab = Object.keys(this.tabs)[0]
+    },
+
+    docSearch (model) {
+      const keys = []
+
+      if (this.search.length) {
```

Comment:
tem mt cascata, nesse caso seria melhor assim: `if (!this.search) return` ai faz a logica embaixo

Reply:
feito